docs/how-tos/create-w3c-document.md

```diff
@@ -0,0 +1,483 @@
+    const baseDocument = {
+      "@context": [
+        "https://trustvc.io/context/bill-of-lading.json",
+        "https://trustvc.io/context/attachments-context.json",
+      ],
+    };
+
+    const document = new DocumentBuilder(baseDocument);
+
+    // Add transferable record configuration
+    document.credentialStatus(credentialStatus);
+    // Add the actual document content/data about the asset
+    document.credentialSubject({
+      BillOfLading: {
+        billOfLadingName: "name",
+      },
+    });
+    // Set when this document expires
+    document.expirationDate(expirationDate);
+    // Define how the document should be rendered visually (template and renderer)
+    document.renderMethod({
+      id: "https://generic-templates.tradetrust.io",
+      type: "EMBEDDED_RENDERER",
+      templateName: "BILL_OF_LADING"
+    }); 
+    
+    return document;
+  } catch (error) {
+    console.error('Error creating document:', error);
+    throw error;
+  }
+};
+```
+
+### 6. Signing the Document
+
+Next, we need to sign our document to create a verifiable credential. We do this by calling the `sign` method on our DocumentBuilder instance and passing the DID key pair. Once signed, the document will include a **proof object** that contains cryptographic evidence of the document's authenticity.
+
+The proof object includes:
+- **type**: The signature algorithm used (e.g., `BbsBlsSignature2020`)
+- **created**: Timestamp of when the signature was created
+- **proofPurpose**: The purpose of the proof (e.g., `assertionMethod`)
+- **proofValue**: The actual cryptographic signature
+- **verificationMethod**: The DID identifier used to verify the signature
+
+This proof allows anyone to cryptographically verify that the document was issued by the holder of the DID and hasn't been tampered with.
+
+Create a new file `src/utils/signW3CDocument.ts` with the signing logic:
+
+```typescript
+require("dotenv").config();
+import { DocumentBuilder } from "@trustvc/trustvc";
+
+export const signDocument = async (document: DocumentBuilder) => {
+  try {
+    const keyPair = process.env.DID_KEY_PAIRS!;
+    const DID_KEY_PAIRS = JSON.parse(keyPair.replace(/\\(?=["'])/g, ""));
+
+    // Sign the document
+    console.log("Signing document...");
+    const signedW3CDocument = await document.sign(DID_KEY_PAIRS);
+    console.log("Document signed successfully", signedW3CDocument);
+
+    return signedW3CDocument;
+  } catch (error) {
+    console.error("Error signing document:", error);
+    throw error;
+  }
+};
+```
+
+### 7. Minting Document for Transferability
+
+Now that we have a signed the W3C verifiable credential, we can **mint** a token that represents ownership of the document. We obtain a tokenId from the signed document and mint it as an NFT on the blockchain.
+
+We'll use TrustVC's **mint method** for minting a token.
+
+Create a new file `src/utils/mintToken.ts` with the minting logic:
+
```

Comment:
Add a sentence to let the user know that this minting is only for ETRs, when they use token registry.